### ISSUE TITLE
Add A License

### ISSUE THREAD

**Issue by Shelob9** [studioraygun/angularwp#4]
I assume this is GPLv2 + or something compatible since you includes WordPress, but can you please state it one explicitly so I can feel comfortable forking for something I'm working on?

**Comment by studioraygun** [comment]
Good point. I'm not really up on licensing so wouldn't know what to classify it as. But feel free to use or redistribute in any way you wish.

Thanks

**Comment by Shelob9** [comment]
Can you just add a notice that it is copyright you, licensed under the terms of the GNU General Public License Version 2 or later?
